tox.ini

```diff
@@ -19,7 +19,6 @@ deps = poetry
 commands =
     poetry install --only testing,main,packages -E all
     poetry run pip install openapi-core==0.13.2
-   	poetry run pip install {toxinidir}[all]
```

Comment:
Seems unnecessary. The first command `poetry install --only testing,main,packages -E all` should install the code as well.